spring-rabbit-stream/src/test/java/org/springframework/rabbit/stream/listener/RabbitListenerTests.java

```diff
@@ -87,16 +97,32 @@ void nativeMsg(@Autowired RabbitTemplate template,
 
 		volatile Context context;
 
-		@Bean
+		@Bean(destroyMethod = "close")
```

Comment:
Do you mean that by default the `close()` method is not inferred by Spring?
I mean this is going to be end-user API anyway to setup an `Environment` for dependency injection, so this concerns me a little bit that `destroyMethod` must be declared explicitly...

Reply:
Sorry, no; that was left over from my experimentation; removing...